### PR TITLE
Fix locator for hosts table

### PR DESCRIPTION
**Pull request by LadislavVasina1**
Test `test_positive_read_details_page_from_new_ui` is failing in the 6.13 automation.
The test was trying to click hostname in the  Hosts table but it was actually clicking on the Actions dropdown in the same row.
Locator change fixes the test failure.
Tested locally.
image

### CODE REVIEW

**Comment by omkarkhatavkar** [comment]
trigger: test-robottelo
pytest: -k test_positive_read_details_page_from_new_ui  tests/foreman/ui/test_host.py

**Review by omkarkhatavkar — APPROVED** [review]

File: airgun/views/host.py [hunk 1]
```diff
@@ -206,7 +206,7 @@ class HostsView(BaseLoggedInView, SearchableViewMixin):
         './/table',
         column_widgets={
             0: Checkbox(locator=".//input[@class='host_select_boxes']"),
-            'Name': Text("./a"),
+            'Name': Text('//*[@id="content"]/table/tbody/tr/td[5]/a'),
```
Comment (shweta83) [inline airgun/views/host.py:209]: Can we try a better locator?Something like 
'Name': Text("//a[contains(@href, '/new/hosts')]")

**Review by shweta83 — CHANGES_REQUESTED** [review]

File: airgun/views/host.py (hunk 1 above)
Comment (shweta83) [inline airgun/views/host.py:209]:
```suggestion
           'Name': Text("//a[contains(@href, '/new/hosts')]"),
```

**Comment by LadislavVasina1** [comment]
trigger: test-robottelo
pytest: -k test_positive_read_details_page_from_new_ui tests/foreman/ui/test_host.py

File: airgun/views/host.py (hunk 1 above)
Comment (LadislavVasina1) [reply airgun/views/host.py]: Thanks for the suggestion, working now.

**Review by vsedmik — APPROVED** [review]

**Comment by LadislavVasina1** [comment]
trigger: test-robottelo
pytest: -k test_positive_assign_taxonomies tests/foreman/ui/test_host.py

**Pull request merged**